crates/provider/src/fillers/signer.rs

```diff
@@ -46,6 +46,10 @@ where
     type Fillable = ();
 
     fn status(&self, tx: &<N as Network>::TransactionRequest) -> FillerControlFlow {
+        if tx.from().is_none() {
+            return FillerControlFlow::Ready;
+        }
+
```

Comment:
wait is this correct? shouldn't this be `is_some()`, "only skip signing by signaling ready if the from is set"

Reply:
if it is none, we are ready for the signer to fill its default